frontend/app/templates/components/user-component/user-subscription.hbs

```diff
@@ -0,0 +1,9 @@
+<h2 class="ui dividing header">Subscription</h2>
+{{#if session.subscription }}
```

Comment:
Where are these properties coming from? I don't see any such relation in the model.

Reply:
The pr is work in progress ;)